app/controllers/posts_controller.rb

```diff
@@ -26,14 +26,18 @@ def edit
   # POST /posts.json
   def create
     @post = Post.new(post_params)
+    pp post_params
     @post.user = current_user
+    @subject = Subject.find(params[:post][:subject])
```

Comment:
This shouldn't be required because we do it below and should replace it with `subject_ids`.